pkg/db/db_local/server_settings.go

```diff
@@ -0,0 +1,121 @@
+package db_local
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/jackc/pgx/v5"
+	"github.com/spf13/viper"
+	"github.com/turbot/steampipe/pkg/constants"
+	"github.com/turbot/steampipe/pkg/db/db_common"
+	"github.com/turbot/steampipe/pkg/utils"
+	"github.com/turbot/steampipe/pkg/version"
+)
+
+type ServerSettingKey string
```

Comment:
does having this as a type really add anything? it's a column key, which intuitively is a string anyway?

Reply:
Just limiting the keys that can be put in